packages/core/services/reflector.service.ts

```
@@ -224,10 +231,10 @@ export class Reflector {
    * @param targets context (decorated objects) to retrieve metadata from
    *
    */
-  public getAllAndOverride<T extends ReflectableDecorator<any>>(
-    decorator: T,
+  public getAllAndOverride<TParam = any, TTransformed = TParam>(
+    decorator: ReflectableDecorator<TParam, TTransformed>,
     targets: (Type<any> | Function)[],
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)